test_app/settings.py

```diff
@@ -23,7 +23,7 @@
 
```

Comment:
If I leave this file unchanged it means there will be no value in `settings.ADMINS`. I have added a check in my code to raise `ImproperlyConfiguredError` if `settings.ADMINS` is not set. I guess we will not be needing that either.